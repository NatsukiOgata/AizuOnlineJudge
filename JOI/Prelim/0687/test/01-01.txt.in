100
697 474 522 967 960 1747 166 1121 49 1919 1959 1704 997 778 1697 688 1604 743 429 144 485 1134 1643 886 542 398 1254 1823 1187 447 1296 287 860 1627 687 342 1615 1517 1098 747 1508 80 1770 574 1845 824 1872 32 83 1343 36 1682 927 742 555 1745 22 987 614 1043 1577 483 727 975 1662 1299 308 1799 1490 182 1754 950 1288 617 1523 1277 1610 98 963 462 1048 692 1847 63 393 790 779 128 15 1248 1198 1743 641 883 769 1994 576 373 87 712
