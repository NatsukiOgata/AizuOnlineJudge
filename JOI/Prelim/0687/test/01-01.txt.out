90805
1748
